### ISSUE TITLE
Bug: Unable to skip when using chromecast

### ISSUE THREAD

**Issue by mLgz0rn** [CherryFloors/jellyfin-plugin-cinemamode#7]
I don't know if this is an issue with jellyfin itself.

Buf if I have 2 trailers, preroll trailer, trailer pre-roll and feature pre-roll enabled, I am only able to skip the first video playing when using chromecast.
If I then skip the first video, when the second stops playing it just returns to the title menu.

So it does only seem to load up the video playing + the next one and if skipped it does not load up the next video to play at all.

**Comment by CherryFloors** [comment]
I believe this is related to this open bug:
https://github.com/jellyfin/jellyfin-android/issues/94

I can't say for certain and don't want to cast blame because there could very well be an issue with the plugin. Because the plugin has been stable with other clients, my instinct is that the intros are being provided by the plugin correctly but mishandled by the casting logic. The intros are similar to a playlist or even chapters in a video. If this is still a problem after the above defect is closed feel free to resubmit.

**Comment by mLgz0rn** [comment]
I was also using the web and android app to chromecast to my NVIDIA shield, and it would skip the first video fine, but if there were more than one if would not.
So say trailer preroll is enabled, 2 trailers and intro, if I skip the trailer preroll it will play the first trailer fine, but if i then skip the first trailer after skipping the trailer preroll it will return to the movie title.

So with having just intro enable it works fine, since the movie is loaded up next.
It's when having more than 2 videos theres an issue.

**Comment by CherryFloors** [comment]
I used the same configuration when testing, trailer pre-roll, 2 trailers, and a feature pre-roll. I was able to skip through all the intros without issue. 

The only thing I can think to suggest is make sure your Jellyfin Android TV client on the NVIDIA shield is up to date (0.15.11 is the latest). An older version of the app crashed when attempting to playback certain trailers when I first started developing this plugin.
